debian/Dockerfile

```diff
@@ -55,6 +55,7 @@ RUN set -ex \
     gpg-agent \
     grub-efi-amd64-bin \
     grub2 \
+    haveged \
```

Comment:
this reduces the startup time of our machines: otherwise docker waits a few minutes for entropy

Reply:
+1